src/shared/components/pluginFilterComponent/pluginFilterComponent.tsx

```diff
@@ -67,7 +67,7 @@ export const PluginFilterComponent = <TMeta extends object, TProps extends objec
     }
 
     return (
-        <div className="flex flex-col gap-6">
+        <div className="flex flex-col gap-2 md:gap-3">
```

Comment:
Adding here since so minor to correct QA on [this ticket](https://aragonassociation.atlassian.net/browse/APP-3868?atlOrigin=eyJpIjoiMjViOWQ1YzJlNmU1NGY5YTg0ZDZiNDM5MDdkZTk3OTEiLCJwIjoiaiJ9) and this is already "fix" PR